SingularityUI/app/templates/tasksTable/tasksActiveBody.hbs

```diff
@@ -4,15 +4,15 @@
             <thead>
                 <tr>
                     <th data-sort-attribute="taskId.id">
-                        Name
+                        Task ID
                     </th>
                     <th class="hidden-xs" data-sort-attribute="taskId.startedAt">
-                        Started
+                        Started At
                     </th>
                     <th class="hidden-xs" data-sort-attribute="host">
                         Host
                     </th>
-                    <th class="hidden-xs" data-sort-attribute="taskId.rackId">
+                    <th class="visible-lg" data-sort-attribute="taskId.rackId">
```

Comment:
Honestly, I just chose the one I saw first as the correct one. There probably won't be room for hosts on a smaller screen if we want to show disk resource too though.